### ISSUE TITLE
Maximum temperature for CO2 EOS

### ISSUE THREAD

**Issue by bonnetr** [usnistgov/REFPROP-issues#69]
Hi all,
Very simple question, I’ve noticed this:
![nist](https://user-images.githubusercontent.com/44962258/48331868-f14e9a80-e651-11e8-8020-12fc081494f0.png)
From 1100K to 2000K is the EOS valid? Typo?
Rgds,
Romain

**Comment by EricLemmon** [comment]
The equations of state for CO2 and several others by Span and Wagner from 20 to 30 years ago were some of the very first equations to be able to extrapolate to much higher pressures and temperatures than could be validated by experimental data.  This was possible by the use of ideal curves, as explained in their paper on CO2.  Below is a picture of isotherms on a pressure vs. density plot.  The red line is 10,000 K, and the highest line is 10,000,000 K.  This shows that the equation is valid far beyond the limits of CO2 as a molecule (i.e., where dissociation occurs), but eventually the equation does show a few unrealistic conditions.  Since that time, we have strived to make our equations valid from near zero kelvin to infinite temperatures, pressures, and densities.  Nearly all equations developed over the last decade are capable of this.  It was originally driven by an attempt to fit an equation of state for helium-3, which is a normal liquid down to about 1 mK or so.  This means that the equation is required to remain valid to less than 1/1000th the value of the critical temperature.  For example, for water, that would mean that the equation would need to work at temperatures less than 1 K (assuming it were a liquid at that state as with helium-3).  During the 15 years since that attempt, we have learned so much that now we have the ability to do this.  Ironically we still have not yet returned to helium-3 to finalize the equation, even though it was the catalyst for the types of equations we use now.  

The other force that pushed us to temperatures far below the triple point is the need for pure fluid equations in mixtures.  For example, mixing nitrogen and ethane requires calculations well below the triple point of nitrogen for very low temperatures where the mixture is still a liquid.  The current nitrogen equation is over 20 years old and does not have the ability to do this, and this has been a problem with much of the current mixture modeling.  We are now in the process of refitting nitrogen to alleviate this problem.

Our current pure fluid equations of state really have no upper or lower limit in T, P, and rho, and could easily extrapolate to the extremely dense fluid phase near the core of Jupiter, for example.  Whether it’s right or not at 10,000,000,000 degrees is surely unknown to us (aside from the fact that the molecules we started with have long since disintegrated - we’re just dreaming of the crazy things we could do).  We do know however, that they don’t work for neutron stars or black holes, but we’re getting there!  So, 2000 K is quite acceptable for CO2, and required for use in mixture models.

![image](https://user-images.githubusercontent.com/42392984/48461573-8c01c300-e791-11e8-8c58-87fb7d27e45c.png)
